storage/cloud.py

```diff
@@ -869,7 +871,10 @@ def __init__(self, context, storage_path, access_key, secret_key, bucket_name):
         # GCS does not support ListObjectV2
         self._list_object_version = _LIST_OBJECT_VERSIONS["v1"]
         upload_params = {}
-        connect_kwargs = {"endpoint_url": GoogleCloudStorage.ENDPOINT_URL}
+        connect_kwargs = {
+            "endpoint_url": GoogleCloudStorage.ENDPOINT_URL,
+            "config": Config(connect_timeout=600, read_timeout=600),
```

Comment:
Is this something that we would want to be configurable?

Reply:
We can certainly add that as well as we did for other storage engines. Sure. I'll make the modifications.

Reply:
it could be very useful to have this as a param